TcBase/TcBase/TcBase/Collection/FB_List.TcPOU

```diff
@@ -0,0 +1,448 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4024.12">
+  <POU Name="FB_List" Id="{3d09efba-b29b-0efb-0296-650e9b624057}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_List IMPLEMENTS I_List
+VAR
+    fbMemMan:       FB_DynMem_Manager;
+    fbBuffer:       FB_DynMem_Buffer(ipMemMan := fbMemMan);
+
+    aPip:           ARRAY [0..1000] OF POINTER TO I_Object; // Debug array

```

Comment:
Might need to find a way to omit debug stuff for release builds in the future

Reply:
https://infosys.beckhoff.com/english.php?content=../content/1033/variant_management/6325752587.html&id=
This might be helpful for that. Will check it out at some point

Reply:
Changes in [1bd793c](https://github.com/ahuca/TcBase/pull/3/commits/1bd793cd22dfac59b452ae03ca35bcfa71e19a46) take care of this aspect